pycbc/__init__.py

```diff
@@ -65,8 +66,10 @@ def sig_handler(signum, frame):
 
     signal.signal(signal.SIGUSR1, sig_handler)
 
+    if level is None:
```

Comment:
I suggest setting `level=logging.INFO` in the function arguments so you can remove this `if` statement.